cmd/pool/order.go

```diff
@@ -401,6 +406,12 @@ var ordersSubmitBidCommand = cli.Command{
 			Name:  "force",
 			Usage: "skip order placement confirmation",
 		},
+		cli.Uint64Flag{
+			Name: "self_chan_balance",
+			Usage: "give the channel leased by this bid order an " +
+				"initial balance by shifting additional funds " +
+				"from our account into the channel",
```

Comment:
Maybe worth mentioning this can be used to open 50/50 balanced channels? 

We'll also need to ensure that the method to ensure enough balance is reserved also adds this component as well.

Reply:
Added the text about 50/50 balanced to the description.
Because we add the self channel balance to the bid's `ReservedValue` method, it is properly included in all balance checks.